GROW

ichen2919 opened the pull request:
This is not exactly an adjective, but I think that it is a word that I
can work with. It is not much now because I am still figuring out how to
execute what I have in mind. I want the letters to expand as the mouse
moves around, but so far it hasn't worked out.

ichen2919 committed:
GROW

This is not exactly an adjective, but I think that it is a word that I
can work with. It is not much now because I am still figuring out how to
execute what I have in mind. I want the letters to expand as the mouse
moves around but so far is hasn't worked out.

LadyK commented:
Hello Isabel,

I like the idea. Can you translate that idea to other forms without text? I ask, because I'd like students to focus on graphics and interactivity, rather than manipulating text and images (which are handled a bit differently in Processing and P5.js). I specified this in the project brief. Please add some comments and notes in your code and give you better direction, if you need that.

LadyK commented:
Hi Isabel. I hope you've been making some progress and just neglected to push it to your repo on github. The project brief specifics for weekly pushes to your repo demonstrating progress. I've been reviewing progress and giving students feedback in order that they can understand what I'm looking for, with reference to this first project. I'm a bit concerned. Please push your progress up to github!

ichen2919 committed:
Adjective

There are still many bugs in this code but after last class I realized
that I should break more things apart into functions especially with the
interactive portions of the project. When the technical parts are
working, I plan on adding more graphics to make it more interesting.

ichen2919 committed:
Adjective

Project
